static/absolute/js/searchPage.js

```diff
@@ -24,14 +26,35 @@ window.onload = function () {
         return;
       }
     } else {
-      // TODO(ved): Use API to convert address --> long/lat
-      alert("This is not yet supported");
-      return;
+      if (/[`!#$%\^&*+=\-\[\]\\';/{}|\\":<>\?]/.test(address)) {
+        alert(
+          "Invalid characters were entered with the address. Please remove them and try again."
+        );
+        return;
+      }
```

Comment:
Do you plan on testing for invalid characters in the back-end? I know that data from the frontend may not always come as we expect.

Reply:
Done.

Reply:
EDIT: All special characters are accepted by the server now. The server encodes the address and uses it for the geocoding api request.